hollow-diff-ui/src/main/java/com/netflix/hollow/HollowDiffUtil.java

```diff
@@ -0,0 +1,17 @@
+package com.netflix.hollow;
+
+import java.text.DecimalFormat;
+
+public class HollowDiffUtil {
+    private static final String[] HEAP_SIZE_UNITS = new String[] { "B", "KB", "MB", "GB", "TB" };
+    private static final DecimalFormat BYTE_FORMATTER = new DecimalFormat("#,##0.#");
+
+    public static String formatBytes(long sizeInBytes) {
+        if (sizeInBytes==0) return "0";
+
+        String sign = (sizeInBytes < 0) ? "-" : "";
+        sizeInBytes = Math.abs(sizeInBytes);
+        int digitGroups = (int) (Math.log10(sizeInBytes)/Math.log10(1024));
```

Comment:
_technically_ this is KiB/MiB etc and not KB/MB (as called out in the comments at https://stackoverflow.com/questions/3263892/format-file-size-as-mb-gb-etc)